docs/concepts/linq.rst

```diff
@@ -0,0 +1,268 @@
+LINQ
+====
+
+What is it?
+-----------
+
+LINQ provides language-level querying capabilities and `higher-order function <https://en.wikipedia.org/wiki/Higher-order_function>`_ API to C# and VB (note that F# already has this capability by nature of being a functional programming language).  It is used either via native query syntax,
+
+.. code-block:: c#
+
+	var dotNetProgrammers = from p in programmers
+	                        where p.IsNewToLINQ
+	                        select new LINQExpert(p);
+	
+Or as a higher-order function via the ``IEnumerable<T>`` interface,
+
+.. code-block:: c#
+
+	var dotNetProgrammers = programmers.Where(p => IsNewToLINQ)
+	                                   .Select(p => new LINQExpert(p));
+									   
+as a way to write expressive, declarative code.
+
+Why is it Useful?
+-----------------
+
+LINQ is an incredibly useful tool for two primary reasons: (1) expressive code, and (2) querying data sources directly from C#/VB via `LINQ Providers <http://stackoverflow.com/a/1568054/1400198>`_  rather than interacting with some obtuse API.
```

Comment:
OF course there is, there is a whole series on implementing LINQ providers (http://blogs.msdn.com/b/mattwar/archive/2007/07/30/linq-building-an-iqueryable-provider-part-i.aspx) and there is a MSDN article @ https://msdn.microsoft.com/en-us/library/vstudio/Bb546158(v=VS.110).aspx.